test/checks/keyboard/page-no-duplicate.js

```diff
@@ -112,7 +112,20 @@ describe('page-no-duplicate', function() {
 			assert.isFalse(check.evaluate.apply(checkContext, params));
 		});
 
-		(shadowSupported ? it : xit)(
+		it('should pass when there are two elements and the first is contained within a nativeSccopeFilter', function() {
+			var options = {
+				selector: 'footer, [role="contentinfo"]',
+				nativeScopeFilter: 'article'
+			};
+			var params = checkSetup(
+				'<article>' +
+					'<footer id="target">Article footer</footer>' +
+					'</article>' +
+					'<footer>Body footer</footer>',
+				options
+			);
+			assert.isTrue(check.evaluate.apply(checkContext, params));
+		})(shadowSupported ? it : xit)(
```

Comment:
What happened here? Prettier artifact?

Reply:
Prettier seems to have done it due to the lack of semicolon. Fixed